pkg/util/config.go

```diff
@@ -104,6 +106,23 @@ func setElement(e reflect.Value, v string) error {
 	return nil
 }
 
+func convertMap(e reflect.Value, v string) error {
+	if e.IsNil() {
+		e.Set(reflect.MakeMap(e.Type()))
+	}
+	vals := strings.Split(v, ",")
+	for _, subitem := range vals {
+		subvals := strings.FieldsFunc(subitem, func(c rune) bool {
+			return c == '<' || c == '=' || c == '>'
```

Comment:
Could you add some documentation or examples around how a map would be specified here? It's not entirely clear what it might look like given these characters you're splitting on. It looks like something like this is a map?

```
x=dog,y=cat,z=bird
```

but also something like

```
x<dog,y>cat,z=bird
```

Am I reading that right?

Reply:
True, you can mix <, > and = in the same string whereas it doesn't look absolutely necessary. Main use-case is to be able to provide parameters such as:
`--extra-config=kubelet.EvictionSoft="memory.available<1%,nodefs.available<1%,imagefs.available<1%" --extra-config=kubelet.EvictionHard="memory.available<1%,nodefs.available<1%,imagefs.available<1%" --extra-config=kubelet.EvictionMinimumReclaim="memory.available=15Mi,nodefs.available=30Mi,imagefs.available=30Mi"`

Reply:
I'm still not sure I understand the need for the < and > separators. Is it in case the value side needs to contain an "=" sign?

Reply:
AFAIU, the value side in the map[string]string only needs to contain the value `1%`, `15Mi`, `30Mi` for instance.
But to stay consistent with kubelet documentation, it make sense to use the same operators:
-  `<` for `eviction-hard` and `eviction-soft` (`--eviction-hard mapStringString                                                                             A set of eviction thresholds (e.g. memory.available<1Gi) that if met would trigger a pod eviction. (default imagefs.available<15%,memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%)`)
-  `=` for `eviction-minimum-reclaim` (`--eviction-minimum-reclaim mapStringString                                                                  A set of minimum reclaims (e.g. imagefs.available=2Gi) that describes the minimum amount of resource the kubelet will reclaim when performing a pod eviction if that resource is under pressure.`)